src/app/navbar/navbar.component.html

```diff
@@ -16,6 +16,10 @@
     <div class="collapse navbar-collapse menu" id="navbarSupportedContent" [collapse]="isCollapsed">
         <ul class="navbar-nav mr-auto">
         </ul>
+        <span class="navbar-text">
+            <img src='http://openweathermap.org/img/w/{{icon}}.png' alt='Icon depicting current weather.'>
```

Comment:
We will want to fix this styling as well, it's making the navbar too tall/fat and has weird whitespace

Reply:
![image](https://user-images.githubusercontent.com/17267100/57038155-18078a00-6c0e-11e9-8947-b60996b3d620.png)
Also let's round the degree so it looks like a user would expect